packages/web-storage/src/index.ts

```diff
@@ -0,0 +1,253 @@
+import cookie from 'js-cookie';
+
+import localStorageInstance from './core';
+
+import { BUCKETS, DEFAULT_STORAGE_EXPIRY_TIME, STORAGE_TYPE } from './constants';
+
+import { checkForErrors, getBucketNameFromKey, getFullKeyForItem, isEmpty } from './helpers';
+
+
+/**
+ * Get Data from Storage
+ *
+ * @remarks
+ * This method is a part of our Storage Library.
+ *
+ * @param key - The Key corresponding to which data is to be extracted.
+ * @param storageType - `{ default: "localStorage" }` The type of storage you want to
+ * extract data from.
+ * @param {string} [bucket='OTHERS'] - `{ default: "OTHERS" }` The bucket in which you want to store data.
+ * We have persisted, auth and others
+ *
+ * @returns The Data stored in Storage if any, else null.
+ *
+ */
+
+export function getDataFromStorage(key: string, storageType: string, bucket = BUCKETS.OTHERS): string | Object | null {
+  const bucketKey = getFullKeyForItem(key, bucket);
+
+  if (checkForErrors() || !localStorageInstance.supported()) {
+    return null;
+  }
+
+  if (storageType === STORAGE_TYPE.LOCAL_STORAGE) {
```

Comment:
switch should work here

Reply:
i guess, if else will be more readable and english like. that is why chosed this one